collect-install.rst.x

```diff
@@ -0,0 +1,98 @@
+*********************************
+Collect Installation and Setup
+*********************************
+
+.. _installing-collect:
+
+Installing Collect
```

Comment:
Perhaps it would be helpful to include device requirements or link to the play store section that shows device requirements?

Reply:
maybe i'm dense... i don't see device reqs in the play store entry...
do we have a formal list of reqs?